AI/VCAI/MapObjectsEvaluator.cpp

```diff
@@ -32,10 +32,6 @@ MapObjectsEvaluator::MapObjectsEvaluator()
 				{
 					objectDatabase[CompoundMapObjectID(primaryID, secondaryID)] = handler->getAiValue().get();
 				}
-				else if(VLC->objtypeh->getObjGroupAiValue(primaryID) != boost::none) //if value is not initialized - fallback to default value for this object family if it exists
```

Comment:
why this got removed

Reply:
AI value is now always in subtype

Reply:
but is there now a way to define default value for new subtypes added by mods?

Reply:
Yes. Same way as for any other properties - "base" field.
See example in diff for `config/objects/moddables.json`

Reply:
is it something new? i thing this did not exist back in the day when I added ai value stuff

Reply:
Nope. Always been here.